content/blog/integrating-k8sgpt-to-empower-kubernetes-with-ai-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,415 @@
+20:32:39-889820 INFO     Loading "Llama-2-13b-chat-hf"                                                                                           
+20:32:39-923551 INFO     TRANSFORMERS_PARAMS=                                                                                                    
+{'low_cpu_mem_usage': True, 'torch_dtype': torch.float32}
+
+Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████| 3/3 [00:47<00:00, 15.91s/it]
+20:33:27-988083 INFO     LOADER: "Transformers"                                                                                                  
+20:33:27-989519 INFO     TRUNCATION LENGTH: 4096                                                                                                 
+20:33:27-990434 INFO     INSTRUCTION TEMPLATE: "Custom (obtained from model metadata)"                                                           
+20:33:27-991414 INFO     Loaded the model in 48.10 seconds.                                                                                      
+20:33:27-992350 INFO     Loading the extension "openai"                                                                                          
+20:33:28-164441 INFO     OpenAI-compatible API URL:                                                                                              
+                                                                                                                                                 
+                         http://0.0.0.0:5000                                                                                                     
+                                                                                                                                                 
+Running on local URL:  http://0.0.0.0:7860
+```
+
+The OpenAI compatible API endpoint is hosted on *'http://0.0.0.0.5000'*. 
+
+Apart from this API endpoint URL, the TGW starts running another local web URL, i.e., at *'http://0.0.0.0:7860'*, providing 3 interface modes, *chat*, *default* & *notebook*, for text generation with the local LLM model backend. 
+
+You can start the browser and type this local URL *'http://0.0.0.0:7860'*. You then land to the *Chat* page. Try to ask some question in the *Chat* page by typing some text and clicking **Generate** button. You may notice it’s a bit slower if you serve the model using CPU inference. But everything should work and you will get the response to your question by *AI*.
+
+
+
+
+
+
+![](/img/local-llm.png)
+
+
+### Integrate K8sGPT with Local AI
+
+
+
+Type the following command to configure K8sGPT to use the local API endpoint. Don’t forget to add the *“/v1”* to the API URL. Hit *Enter* when asking for entering OpenAI key.
+
+
+
+```shell
+$ k8sgpt auth add --backend openai --model Llama-2-13b-chat-hf --baseurl http://localhost:5000/v1

```

Comment:
I am confused here...shouldn't it be "localai" as backend rather than "openai" to integrate K8sGPT with LocalAI backend?